### DISCUSSION TITLE
setFocus is crashing after setValue with React 18

### DISCUSSION THREAD

**Discussion by baonv-acbs-com-vn** [react-hook-form/react-hook-form#10691] [Q&A]
Version Number
7.45.0

Steps to reproduce
    useEffect(() => {
    setValue("quantity","0");
    setFocus("quantity",{ shouldSelect: true })
})
Observe error in the console:
Could not consume error: Error: Something is really wrong.

Expected behaviour
setFocus should focus should Select the field.

What browsers are you seeing the problem on?
Chrome

Relevant log output
No response

Code of Conduct
 I agree to follow this project's Code of Conduct

**Comment by CCherry07** [comment]
Hope to have a demo to repeat this problem.

**Reply by buidlbybao** [reply]
https://codesandbox.io/s/react-hook-form-js-forked-9swm79?file=/src/App.js

**Reply by buidlbybao** [reply]
@CCherry07 My problems

**Reply by buidlbybao** [reply]
setTimeout(function () {
            setFocus('quantity', {shouldSelect: true})
        }, 500);


I'm using setTimeout to handle this my problem! Do you have a better way to handle this problem? tks
